### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by matthiasto**
- adding intro (basically a copy of the one in avafrme.org)
- adding details to the develop section (how to install dependencies...). We need to test if this workflow (installation and running the routine from outside Avaframe works on every linux distribution)
- adding a get started example (a bit as a tutorial)
- moving the asci and shp file routines to in2Trans and all other tools to in3Utils

You can have a look at the hidden doc [here](https://docs.avaframe.org/en/runopedoc/index.html)

### CODE REVIEW

**Comment by awirb** [comment]
I suggest to make a separate Chapter: Installation and move how to install from Develop Avaframe to there

File: docs/develop.rst [hunk 1]
```diff
@@ -13,17 +11,44 @@ Clone repository::
...
+
+    conda create --name avaframe_env
+
+Don't forget to activate it before going on::
+
+    conda activate avaframe_env
+
+Move to ``Avaframe/`` repository and install the different modules::
+
+    conda install --name avaframe_env --file avaframe_env_spec.txt
```
Comment (awirb) [inline docs/develop.rst:36]: I think installation of make is missing

File: docs/develop.rst [hunk 2]
```diff
@@ -13,17 +11,44 @@ Clone repository::
...
+
+Or the equivalent pip commands.
+
+Finally install avaframe::
+
+  conda develop .
```
Comment (awirb) [inline docs/develop.rst:42]: conda develop, conda-build missing?

**Review by awirb — CHANGES_REQUESTED** [review]
Specific text edits in next commit

File: docs/getStarted.rst [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+================
+
+Even-though most of the routines in Avaframe are automatized, running them still
+requires a minimum amount of python scripting.
+Here we provide an example on how to perform simulations with Avaframe,
+but let's start with some information on the utilities to create logs and set the desired configurations.
+
+Utilities
+----------
+
```
Comment (fso42) [inline docs/getStarted.rst:11]: This is fine, but I suggest we have a 'in 5 minutes' (or similiarly named) section at the beginning, in which we have an enumerated list with the absolute bare minimum command and only the absolute necessary explanations to run a simulations. I.e.: 

- Clone the DFA repo by git clone...
- Clone repository by git clone...
- Change to directory i.e cd AvaFrame
- Local install by pip install ...
- Edit avaframe com2DFACfg.ini to 

and so on

File: docs/installation.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+* numpy
+* matplotlib
+* pyshp
+* scipy
+* cmocean
+* seaborn
+* pytest
+
+If you are working on Linux, we suggest to work with anaconda (which you can install following `these instructions <https://docs.anaconda.com/anaconda/install/linux/>`_)
```
Comment (fso42) [inline docs/installation.rst:23]: mention miniconda as well (which I would suggest as default)

File: docs/installation.rst [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+    conda create --name avaframe_env
+
+Don't forget to activate it before going on::
+
+    conda activate avaframe_env
+
+Move to ``Avaframe/`` repository and install the different modules by using the provided installation setup file::
+
+    conda install --name avaframe_env --file avaframe_env_spec.txt
```
Comment (fso42) [inline docs/installation.rst:36]: see main comment

File: docs/installation.rst [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+In order to build the documentation you need to install make ::
+
+  sudo apt install make
+
+Then go to the ``docs\`` directory and run::
+
+  make html
+
+Html files of the documentation can be found in the _build directory.
```
Comment (fso42) [inline docs/installation.rst:71]: I think this should stay in the develop sections. Only someone developing will most likely work on this, all others can look at the online docs...

File: docs/moduleConfiguration.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+The ``getModuleConfig`` function reads the settings from the configuration file (``tmpEx.ini``
+in our example) and writes these settings to the log file. The default settings can be found in the
+configuration file provided within each module.
+It is possible to modify these settings by creating a copy of the configuration
+file (``local_`` followed by the name of the original configuration file) and set the
+desired values of the individual parameters. If there is a configuration file with the
+prefix ``local_``, the ``getModuleConfig`` function reads the settings from this file.
+Another option is to read the settings from a different configuration file, therefore the path to this file
+has to be passed to the ``getModuleConfig`` function directly.
```
Comment (fso42) [inline docs/moduleConfiguration.rst:24]: The order of ini files should be made clearer, i.e. Override -> local_... -> default.ini

**Review by fso42 — CHANGES_REQUESTED** [review]
- I think we should not use a file like avaframe_env_spec unless we really absolutely have to! This requests explicit versions, and I think we should avoid any version setting unless we run into problems. Furthermore this requires us to update requirements and this file and keep them in sync. Apparently conda also allows for installation from a requirements.txt file, we should try this first. 

> conda install --file requirements.txt
> or here
> https://www.technologyscout.net/2017/11/how-to-install-dependencies-from-a-requirements-txt-file-with-conda/
> 

- We need to note which Linux versions this was tested on

File: docs/installation.rst (hunk 6 above)
Comment (awirb) [reply docs/installation.rst:71]: moved to develop.rst

File: docs/getStarted.rst (hunk 3 above)
Comment (awirb) [reply docs/getStarted.rst:11]: Included

**Review by fso42 — APPROVED** [review]

**Review by awirb — APPROVED** [review]

**Pull request merged**